mvn clean install error

[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal com.spotify:docker-maven-plugin:0.3.258:build (default) on project users-microservice: Exception caught: java.util.concurrent.ExecutionException: com.spotify.docker.client.shaded.javax.ws.rs.ProcessingException: org.apache.http.conn.HttpHostConnectException: Connect to localhost:2375 [localhost/127.0.0.1, localhost/0:0:0:0:0:0:0:1] failed: Connection refused -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
[ERROR]
[ERROR] After correcting the problems, you can resume the build with the command

[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal com.spotify:docker-maven-plugin:0.3.258:build (default) on project users-microservice: Exception caught: Tag 8 not found in repository docker.io/library/java -> [Help 1]

--- a/consul-microservice/pom.xml
+++ b/consul-microservice/pom.xml
@@ -9,7 +9,7 @@
     <groupId>org.kbastani</groupId>
     <properties>
         <docker.image.prefix>kbastani</docker.image.prefix>
-        <docker.plugin.version>0.3.258</docker.plugin.version>
+        <docker.plugin.version>0.4.13</docker.plugin.version>
     </properties>
     <dependencies>
         <dependency>

--- a/pom.xml
+++ b/pom.xml
@@ -31,7 +31,7 @@
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <java.version>1.8</java.version>
-        <docker.plugin.version>0.3.258</docker.plugin.version>
+        <docker.plugin.version>0.4.13</docker.plugin.version>
     </properties>
 
     <dependencies>

